### DISCUSSION TITLE
Linux: redirect all youtube links at Freetube ?

### DISCUSSION THREAD

**Discussion by vincegre** [FreeTubeApp/FreeTube#3153] [Q&A]
Hi

I'm using Freetube on Linux (Debian 11). Is there a way to get all youtube links (whatever it comes from web browser, link in email or in instant messaging apps) to open straight in freetube ?

Thanks

Vincèn

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @vincegre, i think that you are looking for something like this https://github.com/FreeTubeApp/FreeTube#browser-extension

**Reply by vincegre** [reply]
> Hi @vincegre, i think that you are looking for something like this https://github.com/FreeTubeApp/FreeTube#browser-extension

Thanks a lot, tried it but not working really well, popup that have to be clicked each time in the browser and opening Freetube but nothing else happens :( Will try to investigate to find out the issue :/

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
You must enable the option within the advanced settings of the extension for it to work

**Reply by absidue** [reply]
@vincegre You might have to download one of the nightly builds, as some desktop environments pass in the URL differently, the nightly builds are able to open those URLs now.
see https://github.com/FreeTubeApp/FreeTube/pull/3119 for more information

Information on how to download the nightly builds are here: https://docs.freetubeapp.io/development/nightly-builds#downloading-nightly-builds

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@vincegre do you think that your questions have been answered?

**Reply by vincegre** [reply]
@efb4f5ff-1298-471a-8973-3d47447115dc I think so, I didn't get really time for now to test it but investigate and reopen a ticket later if needed :) thanks @absidue
